### PR TITLE
Blue feature

### PR DESCRIPTION
**Pull request by rolandoam**
adding new tips and other things

### CODE REVIEW

**Comment by chaqke** [comment]
spam comment

File: tipsAndTricks.md [hunk 1]
```diff
@@ -35,5 +35,66 @@ git blame -M -C -C
...
+2015-11-05	89abb1b	origin/master
+2015-11-05	89abb1b	origin/orange-feature
+2015-11-05	89abb1b	origin/red-feature
+2015-11-05	89abb1b	master
+2015-11-04	ef46177	origin/reveal
+```
+
+- add new files to stash
+```
+git add ______;git stash
```
Comment (ujjwal) [inline tipsAndTricks.md]: `____` could this be renamed as example branch name?

**Comment by ujjwal** [comment]
:+1: except the mentioned comment.

**Comment by erich-unity** [comment]
Please fix the merge conflix.  @chaqke

**Comment by rolandoam** [comment]
@erich-unity merge conflict resolved

**Pull request merged**